quotes/user-popups.css

```diff
@@ -20,7 +20,7 @@
     transition: 0.2s;
     opacity: 1;
     pointer-events: unset;
-    left: 8vw;
+    /*left: 8vw;*/
```

Comment:
:thonk:

Reply:
This needs a second look but it fixes the positioning of the popups on the quotes page to be centered instead of partially off the page.

Reply:
Hmm, this may exist because on the quotes count page, I deliberately chose to have the popup be to the side instead of centered because otherwise it would obstruct the count.
You should check if the popups are centered on the count page, and probably add that specific CSS back for the count page exclusively.